webpack/ForemanTasks/Components/TasksDashboard/Components/ScheduledTasksCard/ScheduledTasksCard.scss

```diff
@@ -5,17 +5,20 @@
   }
   .scheduled-data {
     cursor: pointer;
-    margin-top: 50px;
     font-size: 40px;
     font-weight: 300;
+
+    transition: all 200ms ease-in;
```

Comment:
`transition: font-weight 200ms ease-in;` would make more sense because you are only changing the font-weight

Reply:
The animation feels to slow to me

Reply:
Changed to 50ms

Reply:
I agree with @sharvit